sedlex.opam

```diff
@@ -15,6 +15,7 @@ depends: ["jbuilder" {build}
           "ocaml-migrate-parsetree"
           "gen"
          ]
+depopts: ["menhir"]
```

Comment:
I know little about opam, but I was under the impression that optional dependencies were not trendy these days.  (In particular, if menhir is installed after sedlex, one would need to recompile the whole sedlex packing only the get the extra module.)  Wouldn't it be better to ship the new module as a separate opam package?

Reply:
Fixed.
Though I like the idea that opam would auto rebuilt this package for new dependency menhir. I think not everybody would like to search and install another package for just a few lines of code (but it is fair that rebuilt may cost especially if some large package would depend on this)